safe/common/tables.py

```diff
@@ -201,7 +202,8 @@ class TableRow(object):
     Reference: http://www.w3.org/tr/html4/struct/tables.html#h-11.2.5
     """
 
-    def __init__(self, cells=None, bgcolor=None, header=False, attribs=None,
+    def __init__(
+            self, cells=None, bgcolor=None, header=False, attribs=None,
```

Comment:
Again one item per line policy would apply here, though this is third party code so not as critical